submissions/BogdanGapon/HTML & CSS Practice/sass/layout/_header.scss

```diff
@@ -0,0 +1,195 @@
+.header {
+padding:20px;
+background-color: #fff;
+}
+
+
+.header-nav__list {
+display: flex;
+justify-content: flex-end;
+align-items: center;
+gap: 25px;
+
+}
+
+.header-nav__item  {
+display: flex;
+justify-content: center;
+align-items: center;
+flex-basis: 30px;
+
+}
+
+
+
+.header-nav__link {
+display: flex;
+justify-content: center;
+    align-items: center;
+text-decoration: none;
+color: rgb(17, 15, 15);
+transition: 250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
+
+
+
+&:hover,
+&:focus { 
+color: rgb(170, 63, 63);
+
+}
+
+&.link-img:hover {
+box-shadow: 2px 2px 8px 2px rgba(18, 20, 22, 0.14);
+}
```

Comment:
This is an example of why you should add correct indenations - it is unclear where these two `}` came from. What block is closed here? Indentation would help a lot here.